### PR TITLE
CS-832: Update workflows

### PR DESCRIPTION
**Pull request by rvdh**

### CODE REVIEW

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    types:
+      - published
+
+jobs:
+  publish:
+    name: Publish regular version
+    if: "!contains(github.ref, 'beta')"
+    runs-on: ubuntu-latest
+    env:
+      CLOUDSMITH_TOKEN: ${{ secrets.CLOUDSMITH_TOKEN }}
```
Comment (ChrisTerBeke) [inline .github/workflows/publish.yaml:13]: I think we publish our non-private packages to NPM instead of CloudSmith.

Comment (ChrisTerBeke) [reply .github/workflows/publish.yaml:13]: We could change that however? Not sure what the other @Ultimaker/stardust_frontend peeps think.

Comment (rvdh) [reply .github/workflows/publish.yaml:13]: Ah right, for now I'll add a distinction between the NPM and CloudSmith packages.

Comment (daansmit) [reply .github/workflows/publish.yaml:13]: I do not know enough about the public packages that Ultimaker has to judge this to be honest.

**Comment by rvdh** [comment]
@ChrisTerBeke @daansmit I've added an empty 'build' script to the package.json so we can reuse the same workflow file for all projects. I also added a .npmrc which configures where the package will be published to.

**Pull request merged**